scara:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  

  # Position Controllers ---------------------------------------
  joint1_position_controller:
    type: effort_controllers/JointPositionController
    joint: scara_joint1
    pid: {p: 80.0, i: 0.00, d: 8.0}

  joint2_position_controller:
    type: effort_controllers/JointPositionController
    joint: scara_joint2
    pid: {p: 50.0, i: 0.00, d: 5.0}

  joint3_position_controller:
    type: effort_controllers/JointPositionController
    joint: scara_joint3
    pid: {p: 3000.0, i: 0.00, d: 100.0}

  joint4_position_controller:
    type: effort_controllers/JointPositionController
    joint: scara_joint4
    pid: {p: 0.001, i: 0.00, d: 0.0001}